.changeset/proud-taxis-admire.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sveltejs/adapter-cloudflare-workers': patch
+'@sveltejs/adapter-cloudflare': patch
```

Comment:
```suggestion
'@sveltejs/adapter-cloudflare-workers': minor
'@sveltejs/adapter-cloudflare': minor
```

Reply:
I am ok with this, I don't have an opinion on whether it is a "fix" or "minor".